FieldOpt/ERTWrapper/eclgridreader.h

```diff
@@ -138,27 +130,18 @@ class ECLGridReader
   int ActiveCells();
 
   /*!
-   * \brief GetGridCell get a Cell struct describing the cell with the
```

Comment:
This brief is correct regarding GetGridCell below, should be included there, not?